AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.spec.ts

```diff
@@ -708,8 +708,8 @@ describe('AssignJudgeComponent', () => {
             expect(staffMember.display_name).toBe(newDisplayName);
         });
 
-        describe('changeIsStaffMemberValid', () => {
-            it('should subscribe to AddStaffMemberComponent and update if the staff member is valid', () => {
+        xdescribe('changeIsStaffMemberValid', () => { 
```

Comment:
If you know that a test is no longer needed it should be removed rather than disabled. One less thing to maintain.